raiden/blockchain/abi.py

```diff
@@ -67,6 +68,19 @@ def get_eventname_types(event_description):
     return name, encode_types
 
 
+#  Initially, I wanted to use solc's AST output, but that thing is so crazy that parsing
+#    it would require writing a new module to get just one solidity variable.
+#  So I opted for good ol' regex..
+def parse_contract_version(contract_file):
+    contract_file = get_contract_path(contract_file)
+    version_re = r'^\s*string constant public contract_version = "(\d+.\d+.[\d+\_])";\s*$'
```

Comment:
The regex needs fixing:
- escape the dots
- change `[\d+_]` to `[0-9_]+`

Reply:
thanks for pointing out `\d`. I've always thought that it is equivalent to `[0-9]`, but today after checking SO I learnt it apparently matches all sorts of weird utf-8 chars (insert surprised_emoji.jpg here)